msg/tools/uorb_rtps_message_ids.yaml

```diff
@@ -246,36 +246,54 @@ rtps:
   - msg: collision_constraints
     id: 107
     send: true
+  - msg: orbit_status
+    id: 108
+  - msg: power_monitor
+    id: 109
+  - msg: landing_gear
+    id: 110
   # multi topics
-  - msg: actuator_control0
+  - msg: actuator_controls_0
     id: 120
-  - msg: actuator_control1
+    alias: actuator_controls
+  - msg: actuator_controls_1
     id: 121
-  - msg: actuator_control2
+    alias: actuator_controls
```

Comment:
@bkueng how do you suggest we do that?

Reply:
We have that information in the `.msg` files. I have not looked into how to do it.

Reply:
You have that information on the base type msg msg, that's true. But if you just want to send or receive the alias msg and you actually don't use the base type msg, you can't infer about the message definition of the alias msg. I took this approach because I found it to be the approach that could handle all the cases correctly.

Reply:
Let's keep this approach for now as for CMake is more clear what files to look for and to run the generation scripts in.